docs/using-modules.rst

```diff
@@ -190,5 +197,11 @@ So, exporting external functions from modules is accomplished using the ``export
         ownable_2step.accept_ownership,
     )
 
+    # export all IERC20 functions from `base_token`
+    exports: base_token.IERC20
+
     # export all external functions from `ownable_2step`
     exports: ownable_2step.__interface__
+
+.. note::
+    Any exported interfaces must be implemented by the module. For example, in the above example, ``base_token`` must ``implement`` ``IERC20``, or else the compiler will raise an error.
```

Comment:
typo I think, should be ``implements``. So this wording implies that the exports of interfaces via a module requires an `implements` statement in the module, which is however not true. Or am I missing something?

Reply:
that's exactly what it means

Reply:
re. typo: https://github.com/vyperlang/vyper/pull/4087/commits/7be05e949ae0330b4f71eeef96698067d2366eb3